app/src/main/java/com/example/habitsmasher/ui/dashboard/RecyclerTouchListener.java

```diff
@@ -4,6 +4,7 @@
 import android.animation.ObjectAnimator;
 import android.app.Activity;
 import android.graphics.Rect;
+import android.os.Handler;
```

Comment:
Changes made in this file are to allow for the long press listener to function using the RecyclerTouchListener.